src/commands/synthetics/run-tests-lib.ts

```diff
@@ -150,8 +150,7 @@ export const executeTests = async (
       trigger,
       tests,
       {
-        failOnCriticalErrors: config.failOnCriticalErrors,
-        failOnTimeout: config.failOnTimeout,
+        ...config,
```

Comment:
Nit: could we be more explicit here? It's not obvious what's used from `config`.
To shorten a bit this part, we could destructure `config` before:
```typescript
const {datadogSite, failOnCriticalErrors, failOnTimeout, maxPollingTimeout, subdomain} = config
const results = await waitForResults(
      api,
      trigger,
      tests,
      {datadogSite, failOnCriticalErrors, failOnTimeout, maxPollingTimeout, subdomain},
      reporter,
      tunnel
    )

```